package/MDAnalysis/lib/distances.py

```diff
@@ -61,14 +61,22 @@
 you wish to use is covered by distopia. For more information see the
 `distopia documentation`_.
 
-.. Table:: Functions available using the `distopia`_ backend.
-    :align: center
-
-    +-------------------------------------+-----------------------------------+
-    | Functions                           | Notes                             |
-    +=====================================+===================================+
-    | MDAnalysis.lib.distances.calc_bonds | Doesn't support triclinic boxes   |
```

Comment:
What about the note below about triclinic boxes?

Reply:
Sorry, missed that, fixed.